Tests/test_image_entropy.py

```diff
@@ -0,0 +1,23 @@
+from helper import unittest, PillowTestCase, hopper
```

Comment:
And this should help:

```suggestion
from .helper import unittest, PillowTestCase, hopper
```

Reply:
Thanks also for this – the imports are now relative in the latest force-push.